pkg/kubelet/pluginmanager/operationexecutor/operation_generator.go

```diff
@@ -112,11 +113,14 @@ func (og *operationGenerator) GenerateRegisterPluginFunc(
 		}
 		// We add the plugin to the actual state of world cache before calling a plugin consumer's Register handle
 		// so that if we receive a delete event during Register Plugin, we can process it as a DeRegister call.
-		actualStateOfWorldUpdater.AddPlugin(cache.PluginInfo{
+		err = actualStateOfWorldUpdater.AddPlugin(cache.PluginInfo{
```

Comment:
@taragu I forgot, was there a reason we didn't check for error here?

Reply:
I can't think of why. I think I must have missed it.